tcpclient/connection.go

```diff
@@ -65,6 +67,10 @@ func ConnectionInError(c []Connection) bool {
 	return connectionError.isIn(c)
 }
 
+func TCPProcessingTime(c Connection) time.Duration {
```

Comment:
this is an standard getter. Probably GetProcessingTime is then more convenient

Reply:
In fact, you can associate this function to the struct Connection

Reply:
👍